template/before.sh.erb

```diff
@@ -54,5 +54,26 @@ if [[ ${WORKSPACE_FILE+x} ]]; then
   cp $WORKSPACE_FILE .
 fi
 
+JUPYTER_API="<%= context.jupyterlab_switch == "1" ? "lab" : "tree" %>"
+
+<%- if context.classroom == "1" -%>
+module purge
+module load <%= context.version %>
+
+if [[ -n "${OSC_CLASS_FILES+x}" ]]; then
+  # Copy over classroom materials
+  set -x
+  rsync -avz "$OSC_CLASS_FILES" "$HOME/osc_classes/$OSC_CLASS_ID"
```

Comment:
Every time the job runs this rsync will run. So that means any modification to these files by the user in their home directory will be lost. Is this desired?

Reply:
I think the idea is we copy a subdirectory like `~/osc_classes/BIOCHEM5721/instructor_files` where as a student would create `~/osc_classes/BIOCHEM5721/my_own_files` or just use the root directory.

Either way, the instructor may add or modify files as the semester goes on.

Reply:
So if they get their own copy, if they are going to edit those files would that mean that they would need to copy again? And if they will need to copy again because they need write permission to the files, why do the original copy in the first place? Just so the files appear when they launch the app? With this change it seems like every single launch would delete any changes they made to their copy.

Reply:
We should probably just get clarity from the instructor.

Reply:
Spoke with the instructor. Adding the `--ignore-existing` flag to rsync will allow instructors to add new files as the semester progresses while allowing the users to freely edit they're own copies.